app/main.py

```diff
@@ -1 +1,59 @@
-# write your code here
+from __future__ import annotations
+
+
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+
+        self.skills = ["JavaScript", "CSS", "HTML"]
```

Comment:
Expand the list, don't reassign it. You can use `+=` for this. Fix in other places